<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="messagesTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>560</width>
      <height>280</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="messageLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>440</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>320</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="friendIdLineEdit">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Friend ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addFriendButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>60</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Friend</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getFriendIdButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>100</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Friend ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getUserIdButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>140</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Get User ID</string>
    </property>
   </widget>
   <widget class="QListWidget" name="friendsListWidget">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>180</y>
      <width>180</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="versionLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="errorLineEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>370</y>
      <width>340</width>
      <height>30</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="connectionStatusLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorStatusLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>671</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
